src/transport/grpc/mod.rs

```diff
@@ -622,13 +632,260 @@ impl<'a> TryFrom<&'a Interfaces> for NodeData {
     }
 }
 
+/// Store wrapper designed specifically for the grpc connection
```

Comment:
Grpc Store?
```suggestion
/// GrpcStore designed specifically for the grpc connection
```

Reply:
Since it would have been a bit repetitive, I changed it this way
```suggestion
/// Store implementation designed specifically for the grpc connection
```